odw/core/anonymisation/base.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import List, Sequence, Set
+
+from pyspark.sql import DataFrame, functions as F, types as T
+from pyspark.sql.column import Column
+
+
+STAFF_SEED_COL_CANDIDATES: Sequence[str] = (
+    "Staff Number",
+    "PersNo",
+    "PersNo.",
+    "Personnel Number",
+    "Employee ID",
+    "EmployeeID",
+)
+
+
+
+
+
+
```

Comment:
Not sure if it's a UI issue in github, but if there's whitespace here can you remove it pls, i think ruff will reject it